instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Comment from './Comment.js';
+
+function CommentSection(props) {
+    return (
+        <div>
+            {props.comments.map(comment => (
+                <Comment
+                key={comment.id}    
+                username={comment.username}
+                text={comment.text} 
+                />
+            ))}
+            <input type="text" placeholder="Add a comment..." />
+            <p>time here</p>
```

Comment:
Moments.js will be your Huckleberry here. You will want to google and use that library for the backwards counting time since posting the comment